### DISCUSSION TITLE
AutoFill in Android Edge

### DISCUSSION THREAD

**Discussion by VVYRSCH** [Kunzisoft/KeePassDX#1865] [General]
Hoi there,

In the FAQ it says that AutoFill for Edge is only working in "Compatibility mode". How do I enable this? The link in the FAQ to Android dev sadly doesn't really explain it for beginners.

**Comment by J-Jamet** [comment]
Compatibility mode is automatically activated if you have Play Store services active.
https://developer.android.com/reference/android/service/autofill/AutofillService#compatibility-mode

**Comment by VVYRSCH** [comment]
Thanks for the reply. I do have Play Store services active but still AutoFill in my Edge doesn't show up at all. In Firefox it works perfectly.
What may cause it that it doesn't work?
I have a Redmi Note 9 Pro with Android 12 if that's relevant.
